Code/EditorPlugins/Scene/EnginePluginScene/SceneView/SceneView.cpp

```diff
@@ -183,6 +183,17 @@ void ezSceneViewContext::SetCamera(const ezViewRedrawMsgToEngine* pMsg)
         ezVec3 vUp = pOwner->GetGlobalDirUp();
 
         m_CullingCamera.LookAt(vPosition, vPosition + vForward, vUp);
+
+        auto cameraMode = pCameraComponent->GetCameraMode();
+        float fFovOrDim = pCameraComponent->GetFieldOfView();
+        if (cameraMode == ezCameraMode::OrthoFixedWidth || cameraMode == ezCameraMode::OrthoFixedHeight)
+        {
+          fFovOrDim = pCameraComponent->GetOrthoDimension();
+        }
+
+        const float fNearPlane = pCameraComponent->GetNearPlane();
+        const float fFarPlane = pCameraComponent->GetFarPlane();
+        m_CullingCamera.SetCameraMode(cameraMode, fFovOrDim, fNearPlane, ezMath::Max(fNearPlane + 0.00001f, fFarPlane));
```

Comment:
what is this change for ?

Reply:
This is for culling debugging so that it takes the given fov, near and far plane from the camera component for culling, not the values from the editor camera.